src/PlatformServices/Utilities/AppDomainWrapper.cs

```diff
@@ -9,7 +10,7 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices
     /// <summary>
     /// Abstraction over the AppDomain APIs.
     /// </summary>
-    internal class AppDomainWrapper : IAppDomain
```

Comment:
What's the reason to be removing this interface? Not saying is wrong, just trying to understand.

Reply:
No reason for it. This interface is no longer necessary. It didn't have all functionality - and complicated the code for no reason. So removed to simplify. I want to get rid of this class too in future :)